src/effects/VST3/VST3Effect.cpp

```diff
@@ -192,54 +246,53 @@ VST3Effect::VST3Effect(
    std::shared_ptr<VST3::Hosting::Module> module, 
    VST3::Hosting::ClassInfo effectClassInfo)
 
-   : VST3Wrapper(std::move(module), effectClassInfo)
+   : VST3Wrapper(std::move(module), std::move(effectClassInfo))
 {
    using namespace Steinberg;
    Initialize();
 
+   //defaults
+   mSetup.processMode = Vst::kOffline;
+   mSetup.symbolicSampleSize = Vst::kSample32;
+   mSetup.maxSamplesPerBlock = mUserBlockSize;
+   mSetup.sampleRate = 44100.0;
```

Comment:
These steps are done only in one of two constructors.  Is that right?

Reply:
Copy constructor copies 'master' effect's setup and neccessary modifications are applied later in `VST3Effect::RealtimeAddProcessor`